run-tox-env.yml

```diff
@@ -37,6 +37,8 @@ jobs:
             python: '3.8'
           ${{ if startsWith(env_pair.value, 'py39') }}:
             python: '3.9'
+          ${{ if startsWith(env_pair.value, 'py310') }}:
+            python: '3.10'
```

Comment:
Maybe we should add clauses up to 3.12 since we know that will be a thing?

Reply:
👍 done